src/components/Semester.vue

```diff
@@ -348,21 +356,21 @@ export default {
       const _this = this;
       for (let i = 0; i < splitReq.length; i++) {
         if (splitReq[i].indexOf('"') >= 0) {
-          splitReq[i] = 'true';
-        }
-        if ('()/, '.indexOf(splitReq[i]) < 0) {
+          //don't include things like 'permission of instructor'
+          splitReq[i] = '';
+          //remove operator before & parentheses if directly before and after
+          if(splitReq[i] !== '(') {
```

Comment:
is this line a typo? we just set `splitReq[i]` to the empty string, so it will always remove the thing before. also the `else if` will never happen?

Reply:
Definitely a typo but (I think it should be i-1) but now I'm looking back on that whole section again and I'm not sure it actually works?  Regardless I didn't leave clear enough comments to come back to it and understand what's going on so I think I'm going to have to redo this part

Reply:
I think this makes more sense. But would also fail the `("permission of instructor")` case now?

Either way, might not be worth bothering... see comment below :P

Reply:
I looked over the prereqs/coreqs returned from fireroad and I didn't see any of those cases with parenthesis around just `"permission of instructor"`, but now that I think about it there were several cases with parenthesis around strings that would all get cut out (e.g. `("placement exam", "permission of instructor")` would become `()` which isn't evaluable.  It would be easy enough to just cut out all cases of `()` though.